docs/standard/net-standard.md

```diff
@@ -44,6 +44,8 @@ In general, we recommend you to target the *lowest* version of .NET Standard pos
 1. Target the next lower version of .NET Standard and build your project.
 2. If your project builds successfully, repeat step 1. Otherwise, retarget to the next higher version and that's the version you should use.
 
+However, since targeting lower .NET Standard versions introduces a number of support dependencies, if your project targets .NET Standard 1.x, it is [suggested to *also* target .NET Standard 2.0](https://github.com/dotnet/standard/blob/master/docs/versions.md#how-do-i-know-which-net-standard-version-i-should-target). This will simplify the dependency graph for users of your library that run on .NET Standard 2.0 compatible frameworks and it will reduce the number of packages they will need to download.
```

Comment:
it is suggested to also target --> we recommend that you also target 
will simplify --> simplifies
comma after "compatible frameworks"
will reduce --> reduces
will need --> need